README.md

```diff
@@ -508,7 +508,7 @@ source:
     map:
       name: ip
       model: model
-      username: username
+      user: username
```

Comment:
Your are righ, changed it at the wrong location ... sql uses user not username...